src/app/rundown-overview/components/rundown-overview/rundown-overview.component.ts

```diff
@@ -7,15 +7,17 @@ import { Rundown } from '../../../core/models/rundown';
 import { DialogService } from '../../../shared/services/dialog.service';
 import { Color } from "../../../shared/enums/color";
 import { BasicRundownStateService } from '../../../core/services/basic-rundown-state.service'
+import { SubscriptionLike } from 'rxjs'
 
 @Component({
   selector: 'sofie-rundown-overview',
   templateUrl: './rundown-overview.component.html',
   styleUrls: ['./rundown-overview.component.scss']
 })
-export class RundownOverviewComponent implements OnInit {
-
-  public basicRundowns: BasicRundown[]
+export class RundownOverviewComponent implements OnInit, OnDestroy {
+  public basicRundowns: BasicRundown[] = []
+  public isLoading: boolean = true
+  private subscriptions: SubscriptionLike[] = []
```

Comment:
I have noticed you have also used `SubscriptionLike` somewhere else in another PR.
What's the reason for not using `Subscription`?

If it's because you just want the unsubscripe functionality and nothing else, then what's the reason for not using `Unsubscribable` which is where `SubscriptionLike` gets its unsubscribe from?

I'm in favour of just using `Subscription`. It keeps it "logical" simple :)

Reply:
`Subscription` is an implementation. That would mean that everything that we try to put in the subscriptions must implement all public and private methods and properties that `Subscription` has. The reason I went with SubscriptionLike is that it has the `closed: boolean` property. Such that the consumer of the subscription can see if the connection is live. If we don't care about that, then I am for changing it to use `Unsubscribable`.

Reply:
But you are populating it with actual implemetentations of  `Subscription`.

> That would mean that everything that we try to put in the subscriptions must implement all public and private methods and properties that Subscription has. 

If you find yourself manually creating your own subscriptions, then you are doing something wrong. The `Subscription` is created for us by `Subject.subscribe()`.

Semantically, it makes more sense to work on Subscriptions than something that is like Subscriptions.

Reply:
We are still using `ManagedSubscription`, which is currently implementing SubscriptionLike and not Subscription.